images/node/scripts/origin-node-run.sh

```diff
@@ -2,21 +2,12 @@
 
 set -eu
 
+hostetc=${HOST_ETC:-/rootfs/etc}
 conf=${CONFIG_FILE:-/etc/origin/node/node-config.yaml}
-opts=${OPTIONS:---loglevel=2}
```

Comment:
This was how we're passing in items from the systemd environment file (/etc/sysconfig/origin-node). I believe this will break that behavior. I'll try it as you're proposing and see what happens though.